README.Rmd

```diff
@@ -87,72 +87,119 @@ on how to improve the process on our [forum](https://discuss.ropensci.org/) and
 
 # <a href="#editors" name="editors"></a> Editors and reviewers
 
-### Associate editors
+rOpenSci's Software Peer Review process is run our team of staff and volunteer editors and reviewers.
 
-rOpenSci's Software Peer Review process is run by:
+```{r get-airtable-data, warning=FALSE, echo=FALSE}
+if (Sys.getenv("AIRTABLE_API_KEY") == ""){
+  Sys.setenv(AIRTABLE_API_KEY = params$AIRTABLE_API_KEY)
+}
+if (Sys.getenv("AIRTABLE_ID") == ""){
+  Sys.setenv(AIRTABLE_ID = params$AIRTABLE_ID)
+}
+
+at_eic <- airtabler::airtable(base = Sys.getenv("AIRTABLE_ID"),
+                              table = "editor-in-chief-rotation")
+eic <- at_eic$`editor-in-chief-rotation`$select_all()
+
+at_rev <- airtabler::airtable(base = Sys.getenv("AIRTABLE_ID"),
+                              table = "reviewers-prod")
+reviewers <- at_rev$`reviewers-prod`$select_all()
+
+at_guest <- airtabler::airtable(base = "app8dssb6a7PG6Vwj",
+                                 table = "guest-editors")
+guest_editors <- at_guest$`guest-editors`$select_all()
+```
+
+### Editor-in-Chief
+
+```{r eic, echo=FALSE, results='asis'}
+eic$period_start <- as.Date(eic$period_start)
+eic$period_end <- as.Date(eic$period_end)
+today <- Sys.Date ()
+eic_now <- eic [which (eic$period_start <= today & eic$period_end >= today), ]
+eic_name <- eic_now$acting_aic_name [[1]]
```

Comment:
@noamross Note that airtable has column named `acting_aic_name` rather than `acting_eic_name`. Could you please fix that in airtable and then update here on this line?

Reply:
That's the acting Akaike Information Criterion 😉 

Fixed, lmk if anything breaks.